scripts/serie_update.py

```diff
@@ -264,12 +265,9 @@ def extract_source(self):
             / "Src"
         )
         os_cmd(
-            (
-                "cp",
-                "-r",
-                str(stm32cube_drivers_src_path) + "/*.*",
-                str(temp_drivers_src_path),
-            )
+            "cp -r " + str(stm32cube_drivers_src_path) + "/*.* "
+            + str(temp_drivers_src_path),
+            shell=True,
```

Comment:
I understand that this is in fact the only change required, others are cosmetic and not strictly required (should go into a separate commit)